mjpc/tasks/humanoid/interact/contact_keyframe.h

```diff
@@ -0,0 +1,56 @@
+#ifndef CONTACT_KEYFRAME_H
+#define CONTACT_KEYFRAME_H
+
+#include <mujoco/mujoco.h>
+#include <vector>
+#include <nlohmann/json.hpp>
+using json = nlohmann::json;
+
+namespace mjpc::humanoid {
+
+// ---------- Constants ----------------- //
+constexpr int NOT_SELECTED = -1;
```

Comment:
Please rename these constant variables to follow the [Google C++ style guide](https://google.github.io/styleguide/cppguide.html#Constant_Names). For example `NOT_SELECTED` -> `kNotSelectedInteract`.